hca/dss/__init__.py

```diff
@@ -38,19 +38,23 @@ def download(self, bundle_uuid, replica, version="", dest_name="",
         """
         Download a bundle and save it to the local filesystem as a directory.
 
-        `metadata_files` (`--metadata-files` on the CLI) are one or more shell patterns against which all metadata
-        files in the bundle will be matched case-sensitively. A file is considered a metadata file if the `indexed`
-        property in the manifest is set. If and only if a metadata file matches any of the patterns in
-        `metadata_files` will it be downloaded.
-
-        `data_files` (`--data-files` on the CLI) are one or more shell patterns against which all data files in the
-        bundle will be matched case-sensitively. A file is considered a data file if the `indexed` property in the
-        manifest is not set. If and only if a data file matches any of the patterns in `data_files` will it be
-        downloaded.
+        :param str bundle_uuid: The uuid of the bundle to download
+        :param str replica: the replica to download from. [aws,gcp,azure]
```

Comment:
Please remove "azure" and spell out the possible values.

Reply:
removed azure and specified what the replicas mean.